WhaleSpotting/react-app/src/styles/Home.scss

```diff
@@ -0,0 +1,142 @@
+@import "./constants.scss";
+
+.home {
+    padding-bottom: $footerHeight;
+    margin-top: $headerHeight;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    background: $secondaryLight;
```

Comment:
default background is white so I don't _think_ we need this

Reply:
Done

Reply:
removed